api/api.go

```diff
@@ -650,14 +650,17 @@ func (api *Server) GetLogs(
 	ctx context.Context,
 	in *iotexapi.GetLogsRequest,
 ) (*iotexapi.GetLogsResponse, error) {
+	if in.GetFilter() == nil {
```

Comment:
any chance in == nil?


same below

Reply:
https://github.com/iotexproject/iotex-proto/blob/master/golang/iotexapi/api.pb.go#L2423

it is okay